src/constants/TokenLogoLookupTable.ts

```diff
@@ -0,0 +1,37 @@
+import store from 'state'
+
+import { DEFAULT_LIST_OF_LISTS } from './lists'
+
+class TokenLogoLookupTable {
+  dict: { [key: string]: string[] | undefined } | null = null
+
+  createMap() {
```

Comment:
This is initialization. Instead of returning a dict, could you retype it as `initialize(): void`, and have it set a class boolean for whether its been initialized?